README.md

```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
 capture are lost over time, and makes it possible for different tools to operate
 on the same dataset, thus enabling data portability between tools and workflows.
 
-(Taken from the
-[Introduction](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#introduction)
-of the specification document.)
+SigMF signal recordings typically involve a data file (e.g. a binary file of IQ
+or RF samples) and a metadata file containing plain text that describes the data.
+Together these files represent one recording, such as `example.sigmf-data` and
+`example.sigmf-meta`.  Here is a minimal example of a SigMF `.sigmf-meta` file:
+
+```yaml
+{
+    "global": {
+        "core:datatype": "cf32_le"
+        "core:sample_rate": 1000000,
+        "core:hw": "PlutoSDR with 915 MHz whip antenna",
+        "core:author": "Art Vandelay",
+        "core:version": "1.0.0",
+    },
+    "captures": [
+        {
+            "core:sample_start": 0
+            "core:frequency": 915000000,
+        }
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are currently four ways that you can use SigMF:
```

Comment:
suggest "currently" -> "at least", or maybe just "four community-supported projects for using SigMF".

Reply:
Great suggestion, I've made that change